### DISCUSSION TITLE
Any experience with UGREEN HDMI Splitter

### DISCUSSION THREAD

**Discussion by gabrielnegreira** [awawa-dev/HyperHDR#1020] [General]
I just came across the HMDI splitter from UGREEN:

https://www.amazon.com.be/-/en/gp/product/B0B129NNV1/ref=ox_sc_act_title_1?smid=AXZ3JQ1GVFPIF&psc=1

It seems to tick all boxes for me: 4K, 60 FPS, HDR, HDCP, and CEC. And for 23 euros! I don't care about dolby vision as my TV does not support it anyway. 

Does anyone have an experience with it?

**Comment by gabrielnegreira** [comment]
Well, to anyone wondering I went on and purchased it. Did a couple of tests and it seems to work as advertised. I can get 4k at 60fps with hdr and CEC. Netflix content plays as normal so I guess HDCP is working. So far I am really happy with it and recomend to anyone that do not require dolby vision.

**Reply by AaronPorts** [reply]
What is your experience using this splitter now? Did you have handshake issues?

**Reply by gabrielnegreira** [reply]
Yes, I do face handshake issues quite often when the setup is turned on. I would say 1 out of 3 times I need to turn off and on again to have everything working otherwise the graber gets a green screen (and the leds get green). Once the it is working it remains solid, but having to power on and off a few times is inconvenient. Replacing the splitter is on my `to do` list.

**Comment by awawa-dev** [comment]
The biggest problems with the splitters are: DV support (here apart from Ezcoo and HD Fury it's hard to recommend anything else) and stability because some models can "lose" signal from time to time which suddenly leads to HDMI hand-shake procedure. If you don't have such problems with the grabber itself, your experiences will be helpful for other users looking for an inexpensive 4K and CEC-capable HDMI splitter 👍

**Reply by EvoSems** [reply]
I have this splitter. there is occasional handshake issues with DV on my setup where it just goes black. I have to power cycle the splitter to remedy this.

**Reply by EvoSems** [reply]
other than that. no issues.
